R/rstan_config.R

```diff
@@ -215,7 +215,7 @@ rstan_config <- function(pkgdir = ".") {
     }
     cat("#include <exporter.h>",
         "#include <stan/math/prim/mat/fun/Eigen.hpp>",
-        "#include <stan/model/standalone_functions_header.hpp>",
+        "#include <stan/math/prim/meta.hpp>",
```

Comment:
Does this need to wait for StanHeaders 2.26?

Reply:
Nope, I've included a testing workflow to verify that it works with all of the CRAN and 2.26 combinations of rstan and StanHeaders